docs/http/traffic-policy/expressions/macros.mdx

```diff
@@ -7,14 +7,14 @@ also be used in policy expressions. For convenience, the following custom macros
 
 | Name                                                                       | Return Type | Description                                                                                                                                       |
 | -------------------------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [`hasReqHeader(string)`](#hasreqheaderstring)                              | bool        | Returns true or false if the provided header key is present on the request. Header keys must be written in canonical format.                      |
```

Comment:
should we keep these?  i removed as not necessary anymore given we preprocess the keys to lower case, but maybe it's better user experience to provide narrower instructions

Reply:
yeah I say remove, type as crazy as you want, dear users